vite.config.ts

```diff
@@ -0,0 +1,28 @@
+import {defineConfig} from "vite";
+
+import react from "@vitejs/plugin-react";
+
+function manualChunks(id : string) {
+    if (id.includes('node_modules/react')) {
+        return 'react';
+    } else if (id.includes('node_modules/@material-ui')) {
+        return 'material';
+    } else if (id.includes('node_modules')) {
+        return 'vendor';
+    }
+}
+
+// https://vitejs.dev/config/
+export default defineConfig({
+    plugins: [react()],
+    build: {
+        outDir: 'build',
+
+        // https://rollupjs.org/configuration-options/
+        rollupOptions: {
```

Comment:
there is an issue after running in docker that is solved with commenting that part 
do you know how to fix it?
```
material-dcbda5c5.js:1 Uncaught ReferenceError: Cannot access 'Ve' before initialization
    at material-dcbda5c5.js:1:3416
```
run `docker-compose up --build` and open `localhost:8080`

Reply:
This was indeed due to the library chunks, which are now again all in one.
However there are some issues now with react elements after login...